.github/actions/set_up_macos/action.yml

```diff
@@ -14,7 +14,7 @@ runs:
   steps:
     - uses: maxim-lobanov/setup-xcode@v1
       with:
-        xcode-version: '15.0.1'
+        xcode-version: '15.2'
```

Comment:
This snuck in there from an experiment. I am leaving it as I am upgrading the tests and runners in another PR.